cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/tester/TriggerTestCaseEventListener.java

```diff
@@ -77,6 +77,8 @@ public synchronized void onEvent(ScoreEvent scoreEvent) throws InterruptedExcept
                         data.get(EventConstants.SCORE_ERROR_MSG));
                 break;
             case ScoreLangConstants.SLANG_EXECUTION_EXCEPTION:
+                flowFinished.set(true);
+                result = "FAILURE";
```

Comment:
You shouldn't put here failure result. We can get to a result=FAILURE only in cases where we finished the execution correctly with SCORE_FINISHED_EVENT and not in cases of exceptions.